tools/testrunner/testrunner.go

```diff
@@ -212,6 +213,8 @@ func (r *runner) runTests(ctx context.Context, args []string) {
 
 		// Run tests.
 		stdout, err := currentAttempt.run(ctx, args)
+		// Extract prominent alerts from this attempt's output.
+		r.alerts = append(r.alerts, parseAlerts(stdout)...)
```

Comment:
I forgot that the test-runner collects the entire output in memory. I am a bit concerned about memory usage since it would claim memory the tests and dependencies can't use. Could be solved with a stream-based log parser, but I also don't want to blow up scope. Let's keep that in mind for the future.